lld/test/ELF/linkerscript/wildcards.s

```diff
@@ -91,24 +91,31 @@ SECTIONS {
   .text : { *([.]abc .ab[v-y] ) }
 }
 
-## Test a few non-wildcard meta characters rejected by GNU ld.
+## Test a few non-wildcard characters rejected by GNU ld.
 
 #--- lbrace.lds
-# RUN: ld.lld -T lbrace.lds a.o -o out
+# RUN: not ld.lld -T lbrace.lds a.o 2>&1 | FileCheck %s --check-prefix=ERR-LBRACE --match-full-lines --strict-whitespace
+#      ERR-LBRACE:{{.*}}: section pattern is expected
+# ERR-LBRACE-NEXT:>>>   .text : { *(.a* { ) }
```

Comment:
Is it worth adding a case when there is no space between the disallowed character for example `.text : { *(.a*{)` as I understand it (,),{,} are lexed as a single token so the spaces shouldn't matter. 

From reading the line and the tests all having the character separated by spaces, it made me double check that we could catch more than just a single isolated character.
```
if (s.size() == 1 && strchr("(){}", s[0]))
```

Reply:
Thanks for the suggestion. Updated. Changed `s.size() == 1` to `!s.empty()` to be clearer that we just guard again `""` special case.